src/filters/GLProbes/WebGLProbe.ts

```diff
@@ -37,26 +29,20 @@ class WebGLProbe {
 
   /**
    * query browser for WebGL
-   * @returns config object if true
    */
-  queryWebGL() {
-    if (getEnv().isLikelyNode) {
-      return;
-    }
+  queryGL() {
```

Comment:
why renaming WebGL to GL? 
queryWebGL is meant to detect support for WEBGL, i think the old name is good.
GLPrecision change i think is ok, since the constant is stil GL_PRECISION_XXXXX in the WEBGL api.

Reply:
because I saw there is support for GL on node canvas...
so a dev could somehow wire it up with fabric and use a NodeGLProbe

Reply:
Anyways since we need a probe for node I had to rename the base class/folder

Reply:
The npm module is called `gl` but is still the WebGL specs api.